README.md

```diff
@@ -32,19 +28,24 @@ const ceramic = new Ceramic() // connects to localhost:7007 by default
 
 const config: NftResolverConfig = {
   ceramic,
-  subGraphUrls: { // optional, there are defaults for ethereum mainnet (erc721 and erc1155)
-    // CAIP2 ChainID (below is ETH mainnet)
+  chains: {
     'eip155:1': {
-      // Asset namespace
-      erc721: 'https://api.thegraph.com/subgraphs/name/xxx/yyy',
-      // erc721: 'http://localhost:8000/subgraphs/name/aoeu/qjkx' // also works!
-      erc1155: 'https://api.thegraph.com/subgraphs/name/abc/xyz'
+      blocks: "https://api.thegraph.com/subgraphs/name/yyong1010/ethereumblocks",
+      skew: 15000,
+      assets: {
+        erc721: "https://api.thegraph.com/subgraphs/name/sunguru98/mainnet-erc721-subgraph",
+        erc1155: "https://api.thegraph.com/subgraphs/name/sunguru98/mainnet-erc1155-subgraph",
+      },
+    },
+    'eip155:4': {
```

Comment:
Can we say something about what a CAIP-2 chainId is somewhere so that people can learn what this is?